apps/tup-cms/src/taccsite_cms/templates/assets_custom.html

```diff
@@ -34,6 +34,6 @@
 <script src="{% static 'tup_cms/js/modules/breadcrumbs.js' %}" type="module"></script>
 
 <!-- TUP CMS UI Styles -->
-<style>
-    @import url("{% static 'tup_cms/css/tup-cms.css' %}");
-</style>
+<link rel="stylesheet" href="{% static 'tup_cms/css/tup-cms.for-core-styles.css' %}" />
+<link rel="stylesheet" href="{% static 'tup_cms/css/tup-cms.for-core-cms.css' %}" />
+<link rel="stylesheet" href="{% static 'tup_cms/css/tup-cms.for-tup-cms.css' %}" />
```

Comment:
**Purpose.** Fix CSS import to do so into the correct [cascade layers](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_layers).

<details>

If I `@import`, then each `@import` within the imported stylesheets creates a [nested cascade layer](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_layers#precedence_order_of_nested_cascade_layers), but the layers are expected to be top-level.

</details>

Reply:
**Discussion.** I want to load each stylesheet in a layer, so I need not do that for **each** `@import` within **each** stylesheet[^1], but `<link>` does not offer a way.[^2]

[^1]: Like in `apps/tup-cms/src/taccsite_custom/tup_cms/static/tup_cms/css/tup-cms.for-core-styles.css`.
[^2]: "Note: There is no equivalent of the [<link>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link) method of linking stylesheets. Use @import to import a stylesheet into a layer when you can't use @layer within the stylesheet." — [MDN](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_layers#importing_style_sheets_into_named_and_anonymous_layers_with_import)